couchbase/sink_config.go

```diff
@@ -13,20 +13,22 @@ type SinkConfig struct {
 	Password       string
 	BucketPassword string
 	Bucket         string
+	Query          string
 	UsedServices   []gocb.ServiceType
 	WriteMethod    WriteMethod
 
 	KeyExtractor    s.KeyExtractor
 	ExpiryExtractor ExpiryExtractor
 }
 
-func NewSinkConfig(hosts string, username string, password string, bucketPassword string, bucket string) SinkConfig {
+func NewSinkConfig(hosts string, username string, password string, bucketPassword string, bucket string, query string) SinkConfig {
```

Comment:
Please remove the query param from the "constructor" since its an optional parameter